recipes/install.rb

```diff
@@ -21,7 +21,9 @@
 end
 
 group node["kagent"]["certs_group"] do
-  action :create
-  not_if "getent group #{node["kagent"]["certs_group"]}"
+  action :manage
+  append true
+  excluded_members node['hopsmonitor']['user']
```

Comment:
Is this idempotent?

Reply:
Yes. Action 'manage' will not fail if the group doesn't exist or if the user is not member of that group. Contrary to action 'modify' which will fail if the group does not exist